src/utils/index.ts

```diff
@@ -228,11 +228,11 @@ export const getCorrectPos = (startPos:[number,number],endPos:[number,number])=>
     let y2 = Math.max(startPos[1],endPos[1]);
     const width = x2 - x1;
     const height = y2 - y1;
-    if (width < 1) {
-        x2 = x1 + 1;
+    if (width < 20) {
+        x2 = x1 + 20;
     }
-    if (height < 1) {
-        y2 = y1 + 1;
+    if (height < 10) {
+        y2 = y1 + 10;
     }
```

Comment:
can you also make a check for when the pointer is still near the origin of position something like this:
```ts
    if(width<10){
        y1 = y2;
    }
    else if (width < 20) {
        x2 = x1 + 20;
    }
    if(height<10){
        x1 = x2;
    }
    else if (height < 20) {
        y2 = y1 + 20;
    }
```
also can you make this piece of code specific only to the generateRect function as this piece of code is used for other stuff as well

Reply:
overall great just those two changes and I'll merge this